### ISSUE TITLE
Add example of endpoint which shows all logged metrics

### ISSUE THREAD

**Issue by toymachiner62** [siimon/prom-client#460]
As mentioned in [https://github.com/jochen-schweizer/express-prom-bundle/issues/94](https://github.com/jochen-schweizer/express-prom-bundle/issues/94), prom-client should have an example for having an endpoint which returns the logged metrics in the same format that prometheus understands.

**Comment by zbjornson** [comment]
We have https://github.com/siimon/prom-client/blob/master/example/default-metrics.js, are you looking for something different than that?

**Comment by toymachiner62** [comment]
> are you looking for something different than that?

Yes. That example shows how to setup an endpoint that also collects the default metrics. There's no example that shows how to setup an endpoint that does NOT collect default metrics. In my case I want to expose `/metrics` but I only want to be collecting metrics which I define, and not any default metrics.

**Comment by zbjornson** [comment]
How about the other examples like gauge.js, counter.js, histogram-#.js, are those what you're looking for?

https://github.com/siimon/prom-client/blob/074f339914e5d71b5829cd4a949affae23dbc409/example/gauge.js#L10-L22

**Comment by toymachiner62** [comment]
> How about the other examples like gauge.js, counter.js, histogram-#.js, are those what you're looking for?
> 
> https://github.com/siimon/prom-client/blob/074f339914e5d71b5829cd4a949affae23dbc409/example/gauge.js#L10-L22

Not at all. If you look at the link I posted it has a code snippet.

```js
const promClient = require('prom-client')

app.get('/metrics', function(req,res) {
  res.status(200).set('Content-Type', 'text/plain')
  promClient.register.metrics()
    .then(output => res.end(output))
})
```

There's nothing in the prom-client docs about how to do this and there should be.

**Comment by zbjornson** [comment]
Third try, example/server.js:

https://github.com/siimon/prom-client/blob/5c1c68e1eca3d9a47d615e18f130aed7fae7c011/example/server.js#L92-L99
